app/services/csv/case_log_csv_service.rb

```diff
@@ -29,27 +29,22 @@ def label_from_value(value)
     end
 
     def set_csv_attributes
-      @attributes = CaseLog.attribute_names + %w[unittype_sh]
-      replace_csv_id_fields_with_names
-      order_csv_attributes
+      metadata_fields = %w[id status created_at updated_at created_by_name is_dpo owning_organisation_name managing_organisation_name]
+      metadata_id_fields = %w[managing_organisation_id owning_organisation_id created_by_id]
+      intersecting_attributes = ordered_form_questions & CaseLog.attribute_names
+      remaining_attributes = CaseLog.attribute_names - intersecting_attributes
 
-      @attributes -= CSV_FIELDS_TO_OMIT if @user.present? && !@user.support?
-    end
+      @attributes = (metadata_fields + intersecting_attributes + remaining_attributes - metadata_id_fields + %w[unittype_sh]).uniq
+      move_is_inferred_fields
```

Comment:
Would be nice to eliminate these method calls as well

Reply:
Removed the scheme one, have a look, don't know if worth spending more time on this now